src/main/java/org/jenkinsci/plugins/parallel_test_executor/SplitStep.java

```diff
@@ -29,15 +29,18 @@
 
     private boolean estimateTestsFromFiles;
 
-    @DataBoundConstructor public SplitStep(Parallelism parallelism) {
+    @DataBoundConstructor
+    public SplitStep(Parallelism parallelism) {
```

Comment:
The original formatting was intentional FWIW. (Demonstrative rather than descriptive annotation.)

Reply:
In my opinion code should be formatted as similar as possible.
If would be very hard to put such exceptions into auto-formatter rules.

Reply:
Yeah I am not a fan of auto-formatters either. :-) There is a JEP (or proposed JEP?) on the topic. Since many people seem to be positive about it, I have tried to stay out of the way.